frontend/src/components/App.js

```diff
@@ -1,16 +1,50 @@
 import React, { Component } from 'react';
 import './App.css';
+import PaypalButton from './components/PayPalButton';
```

Comment:
Since we don't currently have a great idea of exactly how this is going to be integrated, let's remove all the parts of the component in the App.js file for now and make a block comment in the main component file  demonstrating the usage.